expyfun/stimuli/tests/test_tracker.py

```diff
@@ -72,8 +73,8 @@ def test_tracker_ud():
     tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], 10, np.inf, 1,
                    change_indices=[2])
     tr.respond(True)
-    tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], np.inf, 10, 1,
-                   change_indices=[2], change_rule='trials')
+    tr = TrackerUD(None, 1, 1, [3, 1.5], [1, 0.5], np.inf, 10, 1,
+                   change_indices=[2], change_rule='trials', x_min=0, x_max=3)
```

Comment:
Ooops, it doesn't seem silly at all. I started to do something along those lines and got distracted.